### PR TITLE
Bug fixes

### PR DESCRIPTION
**Pull request by tatva-harpal**

### CODE REVIEW

File: src/containers/Signout/index.js [hunk 1]
```diff
@@ -188,6 +231,64 @@ const SignoutScreen = (props) => {
...
+              <TouchableOpacity
+                onPress={() => {
+                  active === 'incident'
+                    ? setIncident(`${item?.title}`)
+                    : setUndamage(`${item?.title}`);
+                }}>
+                <View style={{ flexDirection: 'row' }}>
+                  <Text style={styles.truckListTextStyle}>{item?.title}</Text>
+                  {active === 'incident' ? (
+                    incident === `${item?.title}` ? (
```
Comment (tatva-denish) [inline src/containers/Signout/index.js:265]: try to manage with some function, it's not good to use ternary within ternary

File: src/containers/StartWork/confirmSign.js [hunk 2]
```diff
@@ -115,81 +167,86 @@ const ConfirmSign = (props) => {
...
-                      shadowColor: 'rgba(0,0,0,0.1)',
-                    }
-                  : null,
-              ]}>
-              <Image
-                source={common.laughing}
-                style={
-                  (styles.MoodSizeStyle,
+                    ? {
+                        backgroundColor: 'rgb(0,85,142)',
```
Comment (tatva-denish) [inline src/containers/StartWork/confirmSign.js:205]: user color constant instead of static

File: src/containers/StartWork/confirmSign.js [hunk 3]
```diff
@@ -115,81 +167,86 @@ const ConfirmSign = (props) => {
...
-                      backgroundColor: 'rgb(0,85,142)',
-                      shadowColor: 'rgba(0,0,0,0.1)',
-                    }
-                  : null,
-              ]}>
-              <Image
-                source={common.sad}
-                style={
-                  (styles.MoodSizeStyle,
+          <View style={{ flexDirection: 'row' }}>
```
Comment (tatva-denish) [inline src/containers/StartWork/confirmSign.js:170]: Avoid inline style, create classes in common files which is using frequently

File: src/containers/AWBJobView/index.js [hunk 4]
```diff
@@ -82,10 +94,16 @@ const JobDetails = (props) => {
     if (currentStatus == 'Switch to Work') {
       currentStatus =
         trackStatus?.length && trackStatus[trackStatus?.length - 2];
+      // console.log('track status:', trackStatus);
+    }
+    if (currentStatus == 'Switch to driver change') {
```
[the remark this answers is not in the document]
Comment (tatva-kashmeera) [reply src/containers/AWBJobView/index.js]: Done

File: src/containers/Home/index.js [hunk 5]
```diff
@@ -198,6 +199,27 @@ const Home = (props) => {
       </TouchableOpacity>
     );
   };
+  const renderStatus = (item) => {
+    let currentStatus = '';
+    if (
+      contextData?.job?.jobDetails &&
+      contextData?.job?.jobDetails?.id == item?.id
+    ) {
+      if (contextData?.job?.status == 'Waiting for start') {
```
[the remark this answers is not in the document]
Comment (tatva-kashmeera) [reply src/containers/Home/index.js]: Done

File: src/containers/Signout/index.js [hunk 6]
```diff
@@ -134,34 +143,48 @@ const SignoutScreen = (props) => {
...
-                ? unauth.checkMarkActiveGreen
-                : unauth.checkMarkGrey
-            }
-            style={styles.CheckMarkSizeStyle}
-          />
-        </TouchableOpacity>
-        <Text style={styles.titleTxt}>{item?.title}</Text>
-      </View>
+      // <View style={styles.listItem}>
+      // <TouchableOpacity
```
[the remark this answers is not in the document]
Comment (tatva-kashmeera) [reply src/containers/Signout/index.js]: Done

File: src/containers/Signout/index.js (hunk 1 above)
Comment (tatva-kashmeera) [reply src/containers/Signout/index.js]: Done

File: src/containers/StartWork/confirmSign.js (hunk 2 above)
Comment (tatva-kashmeera) [reply src/containers/StartWork/confirmSign.js]: Done

File: src/containers/StartWork/confirmSign.js (hunk 3 above)
Comment (tatva-kashmeera) [reply src/containers/StartWork/confirmSign.js]: Done

**Pull request merged**